File: Abundant_num.java
import java.io.*;
import java.util.*;
class Abundant_num
{
	public static void main(String args[])
	{
		int n,b,sum=0;
		Scanner scan=new Scanner(System.in);
		System.out.println("enter the series number>>");
		n=scan.nextInt();
		for(int i=1;i<=n/2;i++)
		{
			if(n%i==0)
				sum=sum+i;

	    }
	    if(sum>n)
	    System.out.println("it is a abundant number sum "+sum +">"+n);
	    else
	    System.out.println("it is a abundant number sum"+sum+"<"+n);
	    	
    }
}

File: abundant_num.py
input_num=int(input('enter the divisors number>'))
divisor_sum=0
for i in range(1,int(input_num/2)+1) :
		if(input_num%i==0) :
			divisor_sum+=i
if divisor_sum>input_num:			
 print('it is abundant number')
else:
 print('it is not a abundant number')			
